lib/matplotlib/widgets.py

```diff
@@ -472,7 +512,13 @@ def _update(self, event):
                event.inaxes != self.ax)):
             self.drag_active = False
             event.canvas.release_mouse(self.ax)
+            self._handle.set_markeredgecolor(self._handle_edgecolor)
+            self._handle.set_markerfacecolor(self._handle_color_released)
+            self.ax.figure.canvas.draw_idle()
             return
+        if event.name == 'button_press_event':
+            self._handle.set_markeredgecolor(self._handle_color_grabbed)
+            self._handle.set_markerfacecolor(self._handle_color_grabbed)
```

Comment:
I'm not sure if this action helpful. If sliders have GUI feedback, it's usually already done on hovering to indicate that you have an control element under the cursor. IMHO changing the color after clicking is too late and serves no purpose.

For simplicity, I'd refrain from changing the color at all.

Reply:
I did this because I was copying the behavior of ipywidgets sliders without much critical thought :grimacing:.

I have a slight preference for them changing color after selection vs not but I'll happily remove it when I next push here. Reducing the number of new configuration kwargs seems like a good thing.

Reply:
That said changing as you mouse over the axes could be nice. I imagine this could be accomplished by using `axes_enter_event` and `axes_leave_event`

Reply:
If you want an effect, see also `Button` for a mouse-over effect. Though here you might want to additionally check if the cursor is on the knob.

We could also say that the "active" color is the same color as the bar (C0). That would still save us the extra argument. One can always add more configurability later when the need arises.

Reply:
> ipywidgets use color to indicate focus (e.g. which control reacts to the keyboard). I don't think our widgets have a concept of focus.


aaah. I'm fully convinced by this, I'll remove it as it's bad to indicate something that is isn't there.